sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/arithmetic.scala

```diff
@@ -62,7 +66,11 @@ case class UnaryMinus(child: Expression) extends UnaryExpression
 }
 
 @ExpressionDescription(
-  usage = "_FUNC_(a) - Returns a.")
+  usage = "_FUNC_(expr) - Returns the positive value of expr.",
```

Comment:
Nit: I would just say "Returns the value of expr". Unary plus doesn't do anything. This sort of sounds like it makes the value positive.